src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -1022,6 +1048,65 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
         this.selectedProjectType = history.state.programmingExerciseForImportFromFile.projectType;
     }
 
+    /**
+     * In case the programming language or project type changes, we need to reset the template and the build plan
```

Comment:
Also, when toggling options like "Sequential Test Runs". When I tested this locally, the build plan did not update

Reply:
I don't think changing the language or project type after the creation is actually possible (for now)
However, it's possible to activate static code analysis or your mentioned sequential test runs.

Reply:
The reloading should work now